terraform/bambictf.tf

```diff
@@ -204,12 +204,12 @@ systemctl start wg-quick@internal
 TERRAFORMEOF
 }
 
-resource "hcloud_server" "moloch" {
+resource "hcloud_server" "elk" {
   name        = "moloch${count.index + 1}"
-  image       = data.hcloud_image.bambimoloch.id
+  image       = data.hcloud_image.bambielk.id
   location    = "fsn1"

```

Comment:
Could you rebase/merge the master branch and use `local.location` here?

Reply:
done

Reply:
sure?

Reply:
yes

Reply:
But you use that for the image instead of the location :D

Reply:
thought you did mean that :D

Reply:
was wondering why but thought yoi would have your reasons

Reply:
no (tm) should be right